### DISCUSSION TITLE
Add TLS at RPC API

### DISCUSSION THREAD

**Discussion by mikromas** [fluent/fluentd#4291] [Q&A] [answered]
Is possible to configure tls at RPC through which user can manage Fluentd instance through HTTP endpoints.

e.g. is possible the below to work somehow?
<system>
  rpc_endpoint 0.0.0.0:24444
  workers 4
  <transport tls>
    cert_path /path/to/server-certificate.pem
    private_key_path /path/to/server-private-key.pem
    ca_path /path/to/ca-cert.pem
  </transport>
</system>

**Comment by kenhys** [comment, marked as answer]
Currently, RPC endpoint is not designed to support TLS directly.
If you want to provide RPC endpoint feature via HTTPS, use reverse proxy and so on.

**Reply by mikromas** [reply]
Thanks for the reply.
